### PR TITLE
NG-1130 POC video quality

### PR DESCRIPTION
**Pull request by euclides-jr**

### CODE REVIEW

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java [hunk 1]
```diff
@@ -905,6 +927,50 @@ public void onVideoTrackDisabled(
...
+    private LocalParticipant.Listener localParticipantListener() {
+        return new LocalParticipant.Listener() {
+            @Override
+            public void onNetworkQualityLevelChanged(@NonNull LocalParticipant localParticipant, @NonNull NetworkQualityLevel networkQualityLevel) {
+                localParticipantNQStatus.setText("Local: " + networkQualityLevel.name());
+            }
+
+            @Override
+            public void onAudioTrackPublished(@NonNull LocalParticipant localParticipant, @NonNull LocalAudioTrackPublication localAudioTrackPublication) {
+
```
Comment (javipascual) [inline android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java:947]: why are these functions needed?

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java [hunk 2]
```diff
@@ -905,6 +927,50 @@ public void onVideoTrackDisabled(
...
+            @Override
+            public void onDataTrackPublished(@NonNull LocalParticipant localParticipant, @NonNull LocalDataTrackPublication localDataTrackPublication) {
+
+            }
+
+            @Override
+            public void onDataTrackPublicationFailed(@NonNull LocalParticipant localParticipant, @NonNull LocalDataTrack localDataTrack, @NonNull TwilioException twilioException) {
+
+            }
```
Comment (javipascual) [inline android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java:973]: where is the quality shown?

Comment (euclides-jr) [reply android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java:973]: onNetworkQualityLevelChanged

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java (hunk 1 above)
Comment (euclides-jr) [reply android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java:947]: Yes, they are part of the interface and required

**Review by javipascual — APPROVED** [review]

**Review by f1shear — APPROVED** [review]

**Pull request merged**